Traceback (most recent call last):
  File "D:/neteasenews/neteasenews/spider/photospider.py", line 105, in <module>
    photospider()
  File "D:/neteasenews/neteasenews/spider/photospider.py", line 101, in photospider
    photo(pic_tabs)
  File "D:/neteasenews/neteasenews/spider/photospider.py", line 49, in photo
    browser.get(tab_url)
  File "C:\Anaconda3\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 324, in get
    self.execute(Command.GET, {'url': url})
  File "C:\Anaconda3\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 312, in execute
    self.error_handler.check_response(response)
  File "C:\Anaconda3\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: 'url' must be a string
  (Session info: headless chrome=65.0.3325.181)
  (Driver info: chromedriver=2.36.540470 (e522d04694c7ebea4ba8821272dbef4f9b818c91),platform=Windows NT 10.0.16299 x86_64)

